docs/sources/alerting/fundamentals/alert-rules/alert-rule-types.md

```diff
@@ -63,15 +63,18 @@ For more information on recording rules in Prometheus, refer to [recording rules
-| Create alert rules based on data from any of our supported data sources | Yes                                                                                                                          | No: You can only create alert rules that are based on Prometheus data. The data source must have the Ruler API enabled.                                 |
-| Mix and match data sources                                              | Yes                                                                                                                          | No                                                                                                                                                      |
-| Includes support for recording rules                                    | No                                                                                                                           | Yes                                                                                                                                                     |
-| Add expressions to transform your data and set alert conditions         | Yes                                                                                                                          | No                                                                                                                                                      |
-| Use images in alert notifications                                       | Yes                                                                                                                          | No                                                                                                                                                      |
-| Scaling                                                                 | More resource intensive, depend on the database, and are likely to suffer from transient errors. They only scale vertically. | Store alert rules within the data source itself and allow for “infinite” scaling. Generate and send alert notifications from the location of your data. |
-| Alert rule evaluation and delivery                                      | Alert rule evaluation and delivery is done from within Grafana, using an external Alertmanager; or both.                     | Alert rule evaluation and alert delivery is distributed, meaning there is no single point of failure.                                                   |
+| Create alert rules<wbr /> based on data from any of our supported data sources | Yes | No: You can only create alert rules that are based on Prometheus data. The data source must have the Ruler API enabled. |
+| Mix and match data sources | Yes | No |
+| Includes support for recording rules | No | Yes |
+| Add expressions to transform<wbr /> your data and set alert conditions | Yes | No |
```

Comment:
If possible, I would keep the semantic phrases together:

"Add expressions to transform your data<wbr /> and set alert conditions"

Otherwise, the expressions are transforming themselves (intransitively) grammatically.